package.json

```diff
@@ -10,11 +10,11 @@
     "flow:build": "flow gen-flow-files ./src/index.js --out-dir ./dist/module",
     "karma": "cross-env NODE_ENV=test babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/karma start",
     "babel": "babel src/ --out-dir dist/module",
-    "webpack": "babel-node ./node_modules/.bin/webpack --progress",
+    "webpack": "babel-node ./node_modules/.bin/webpack",
     "webpack-base": "babel-node ./node_modules/.bin/webpack --progress --config webpack.base.config.js",
     "webpack-button": "babel-node ./node_modules/.bin/webpack --progress --config webpack.button.config.js",
-    "test": "if [ ! $SKIP_TEST ]; then npm run lint && npm run flow-typed && npm run flow && npm run test-ssr && npm run karma && npm run buttondiff; fi;",
-    "build": "npm run test && npm run webpack",
+    "test": "npm run lint && npm run flow-typed && npm run flow && npm run test-ssr && npm run karma && npm run buttondiff",
```

Comment:
should `npm run buttondiff` also be removed from `test`? (will CI continue to hang?)

Reply:
`npm run test` doesn't get run by CI. It's really only there as a convenience script for local development (to be run manually).